plugins/blocks/user/UserBlockPlugin.inc.php

```diff
@@ -61,9 +61,16 @@ function getContents(&$templateMgr) {
 			$templateMgr->assign_by_ref('userSession', $session);
 			$templateMgr->assign('loggedInUsername', $session->getSessionVar('username'));
 			$loginUrl = Request::url(null, 'login', 'signIn');
+			// if the page is not ssl enabled, and force_login_ssl is set, this flag will present a link instead of the form
+			$forceSSL = 0;
```

Comment:
Looks good, but do you mind changing the flag to use true/false rather than numbers?